docs/consume-packages/install-use-packages-powershell.md

```diff
@@ -1,178 +1,205 @@
+- Reverses any changes that installing the package made to *app.config* or *web.config*.
 - Removes previously-installed dependencies if no remaining packages use those dependencies.
 
+To uninstall a package and all its unused dependencies, run:
+
+```powershell
+Uninstall-Package <PackageName> -RemoveDependencies
+```
+
+To uninstall a package even if other packages depend on it, run:
+
+```powershell
+Uninstall-Package <PackageName> -Force
+```
+
 ## Update a package
 
-```ps
-# Checks if there are newer versions available for any installed packages
-Get-Package -updates
+To update a package, use [Get-Package](../reference/ps-reference/ps-ref-get-package.md) and [Update-Package](../reference/ps-reference/ps-ref-update-package.md). You can run the following commands:
 
-# Updates a specific package using its identifier, in this case jQuery
-Update-Package jQuery
+- To check if there are newer versions available for any installed packages:
 
-# Update all packages in the project named MyProject (as it appears in Solution Explorer)
-Update-Package -ProjectName MyProject
+  ```powershell
+  Get-Package -updates
+  ```
 
-# Update all packages in the solution
-Update-Package
-```
+- To update a specific package:
 
-See [Get-Package](../reference/ps-reference/ps-ref-get-package.md) and [Update-Package](../reference/ps-reference/ps-ref-update-package.md)
+  ```powershell
+  Update-Package <PackageName>
+  ```
 
-## Find a package
+- To update all packages in a project:
 
-```ps
-# Find packages containing keywords
-Find-Package elmah
-Find-Package logging
+  ```powershell
+  Update-Package -ProjectName <ProjectName>
+  ```
 
-# List packages whose ID begins with Elmah
-Find-Package Elmah -StartWith
+- To update all packages in the solution:
 
-# By default, Get-Package returns a list of 20 packages; use -First to show more
-Find-Package logging -First 100
+  ```powershell
+  Update-Package
+  ```
 
-# List all versions of the package with the ID of "jquery"
-Find-Package jquery -AllVersions -ExactMatch
-```
+<a name="use-the-nugetexe-cli-in-the-console"></a>
+## Use the NuGet CLI in the console
 
-See [Find-Package](../reference/ps-reference/ps-ref-find-package.md). In Visual Studio 2013 and earlier, use [Get-Package](../reference/ps-reference/ps-ref-get-package.md) instead.
+You can also do all operations that are available in the console with the [NuGet CLI](../reference/nuget-exe-cli-reference.md). However, console commands operate within the context of Visual Studio saved project and solution, and often do more than their equivalent NuGet CLI commands. For example, installing a package through the console adds a reference to the project file, but the NuGet CLI command doesn't. For this reason, developers working in Visual Studio typically prefer to use the console commands rather than the NuGet CLI.
```

Comment:
Given that we give a disclamier later in ther sentence, should we start with you can do `most operations` or something like that?

Reply:
@nkolev92 - Also clarified that this is talking about running the nuget.exe commands in the console, instead of PS.